src/cws_pattern.js

```diff
@@ -182,6 +185,10 @@ function get_zip_name(url, /*optional*/filename) {
     return filename.replace(/\.(crx|jar|nex|xpi|zip)$/i, '') + '.zip';
 }
 
+function get_amo_base_url(amoDomain) {
+    return 'https://' + amoDomain + '/' + (amoDomain=="addons.thunderbird.net" ? "thunderbird" : "firefox")
```

Comment:
Please be consistent in use of `'` and `"`, and whitespace around operators. Change this to:

```javascript
if (amoDomain === 'addons.thunderbird.net') {
    return 'https://addons.thunderbird.net/thunderbird';
}
return 'https://' + amoDomain + '/firefox';
```